Multisig/UI/Safe Management/Safe Settings/SafeSettingsContentView.swift

```diff
@@ -44,6 +47,33 @@ struct SafeSettingsContentView: View {
                     BodyText("Advanced")
                 }
             }
+            
+            Section(header: ListSectionHeader(text: " ")) {
+                Button(action: {
+                    self.showDeleteConfirmation.toggle()
+                }) {
+                    HStack {
+                        Image("ico-remove")
+                        Text("Remove Safe").font(.gnoHeadline)
+                        Spacer()
+                    }
+                    .padding()
+                }
+                .foregroundColor(Color.gnoTomato)
+                .buttonStyle(BorderlessButtonStyle())
+                .background(Color.gnoWhite)
+                .listRowInsets(EdgeInsets(top: 0, leading: 0, bottom: 0, trailing: 0))
+                .actionSheet(isPresented: $showDeleteConfirmation) {
+                    ActionSheet(title: Text(""), message: Text("Removing a Safe only removes it from this app. It does not delete the Safe from the blockchain. Funds will not get lost."), buttons: [
+                        .destructive(Text("Remove")) {
+                            Safe.delete(safe: self.safe)
+                            Safe.selectFirst()
+                            App.shared.state = .balanaces
+                        },
+                        .cancel()
+                    ])
```

Comment:
I would extract the whole button in a separate "RemoveSafeButton" view and move the "showDeleteConfirmation" there as well